src/plugins/stats.ts

```diff
@@ -1,7 +1,7 @@
 import { AnyResponse } from '@octokit/rest'
 import { Request,Response } from 'express'
 
-// Built-in plugin to expose stats about the deployment
+// Built-in app to expose stats about the deployment
```

Comment:
I'm not sure that the stats endpoint id really a plugin, but I definitely wouldn't consider it a built in app.